innovation.game.php

```diff
@@ -8386,6 +8392,54 @@ function stPlayerInvolvedTurn() {
             case "182N1":
                 $step_max = 1;
                 break;
+
+            // id 185, Artifacts age 8: Parnell Pitch Drop
+            case "185N1":
+                // "Draw and meld a card of value one higher than the highest top card on your board."
+                $card = self::executeDraw($player_id, self::getMaxAgeOnBoardTopCards($player_id) + 1, 'board');
+                $count_clocks = 0;
+                if($card['spot_1'] == 6){
+                    $count_clocks++;
+                }
+                if($card['spot_2'] == 6){
+                    $count_clocks++;
+                }
+                if($card['spot_3'] == 6){
+                    $count_clocks++;
+                }
+                if($card['spot_4'] == 6){
+                    $count_clocks++;
+                }                
+                if ($count_clocks == 3) {
+                    // If the melded card has three clocks, you win.
+                    self::notifyPlayer($player_id, 'log', clienttranslate('${You} melded a card with 3 clocks.'), array('You' => 'You'));
+                    self::notifyAllPlayersBut($player_id, 'log', clienttranslate('${player_name} melded a card with 3 clocks.'), array('player_name' => self::getColoredText(self::getPlayerNameFromId($player_id), $player_id)));
+                    self::setGameStateValue('winner_by_dogma', $player_id);
+                    self::trace('EOG bubbled from self::stPlayerInvolvedTurn Parnell Pitch Drop');
+                    throw new EndOfGame();
+                }
+                break;
+
+            // id 189, Artifacts age 8: Ocean Liner Titanic
+            case "189N1":
+                for($color = 0; $color < 5; $color++) {
+                    $card = self::getBottomCardOnBoard($player_id, $color);
+                    if($card != null) {
+                        // Score all bottom cards from your board.
+                        $card = self::transferCardFromTo($card, $player_id, 'score', false, true);
```

Comment:
In order to make it more obvious what that "true" is for, let's add a comment:

$card = self::transferCardFromTo($card, $player_id, 'score', false, /*score_keyword=*/ true);

Reply:
sure.